client/py/synnax/control/controller.py

```diff
@@ -122,9 +122,46 @@ def set(
         ch = retrieve_required(self._retriever, ch)[0]
         self._writer.write({ch.key: value, ch.index: TimeStamp.now()})
 
-    def authorize(self, ch: ChannelKey | ChannelName, value: Authority):
-        ch = retrieve_required(self._retriever, ch)[0]
-        self._writer.set_authority({ch.key: value, ch.index: value})
+    @overload
+    def set_authority(
+        self,
+        value: CrudeAuthority,
+    ) -> bool:
+        ...
+
+    @overload
+    def set_authority(
+        self,
+        value: dict[ChannelKey | ChannelName, CrudeAuthority],
+    ) -> bool:
+        ...
+
+    @overload
+    def set_authority(
+        self,
+        ch: ChannelKey | ChannelName,
+        value: CrudeAuthority,
+    ) -> bool:
+        ...
+
+    def set_authority(
+        self,
+        value: (
+            dict[ChannelKey | ChannelName | ChannelPayload, CrudeAuthority]
+            | ChannelKey
+            | ChannelName
+            | CrudeAuthority
+        ),
+        authority: CrudeAuthority | None = None,
```

Comment:
You have a parameter called `authority` but none of the overloads have `authority` as a parameter.